calculator/src/components/Ouput.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import { Doughnut } from 'react-chartjs-2';
+
+const Ouput = ({data, newKey}) => {
+
+  const [tip, setTip] = useState(data.tip);
```

Comment:
I don't know why you are using useEffect and useState as far as I know you can just use data.billTotal  in the HTML might make your code cleaner.

Reply:
haha yes ... at first it wasn't working because i didnt have data as a state in the parent component so i freaked out and started trying stuff but i figured it out and I was out of time to clean up the code